replica/distributed_loader.cc

```diff
@@ -246,6 +246,60 @@ highest_version_seen(sharded<sstables::sstable_directory>& dir, sstables::sstabl
     });
 }
 
+} namespace sstables {
+
```

Comment:
namespace declaration should start in the next line. Checked it's not specified in the coding style doc (in seastar), but that's what we have been doing throughout our code base.

Reply:
It's temporary, next patch drops this namespace break